ext/imap/php_imap.stub.php

```diff
@@ -5,251 +5,250 @@
+/** @param resource $imap */
+function imap_createmailbox($imap, string $mailbox): bool {}
 
 /**
- * @param resource $stream_id
+ * @param resource $imap
  * @alias imap_createmailbox
  */
-function imap_create($stream_id, string $mailbox): bool {}
+function imap_create($imap, string $mailbox): bool {}
 
-/** @param resource $stream_id */
-function imap_renamemailbox($stream_id, string $old_name, string $new_name): bool {}
+/** @param resource $imap */
+function imap_renamemailbox($imap, string $from, string $to): bool {}
 
 /**
- * @param resource $stream_id
+ * @param resource $imap
  * @alias imap_renamemailbox
  */
-function imap_rename($stream_id, string $old_name, string $new_name): bool {}
+function imap_rename($imap, string $from, string $to): bool {}
 
-/** @param resource $stream_id */
-function imap_deletemailbox($stream_id, string $mailbox): bool {}
+/** @param resource $imap */
+function imap_deletemailbox($imap, string $mailbox): bool {}
 
-/** @param resource $stream_id */
-function imap_subscribe($stream_id, string $mailbox): bool {}
+/** @param resource $imap */
+function imap_subscribe($imap, string $mailbox): bool {}
 
-/** @param resource $stream_id */
-function imap_unsubscribe($stream_id, string $mailbox): bool {}
+/** @param resource $imap */
+function imap_unsubscribe($imap, string $mailbox): bool {}
 
-/** @param resource $stream_id */
-function imap_append($stream_id, string $folder, string $message, ?string $options = null, ?string $internal_date = null): bool {}
+/** @param resource $imap */
+function imap_append($imap, string $folder, string $message, ?string $options = null, ?string $internal_date = null): bool {}
 
-/** @param resource $stream_id */
-function imap_ping($stream_id): bool {}
+/** @param resource $imap */
+function imap_ping($imap): bool {}
 
-function imap_base64(string $text): string|false {}
+function imap_base64(string $string): string|false {}
 
-function imap_qprint(string $text): string|false {}
+function imap_qprint(string $string): string|false {}
 
-function imap_8bit(string $text): string|false {}
-
-function imap_binary(string $text): string|false {}
+function imap_8bit(string $string): string|false {}
 
+function imap_binary(string $string): string|false {}
 
 function imap_utf8(string $mime_encoded_text): string {}
 
 /**
- * @param resource $stream_id
+ * @param resource $imap
  * @return stdClass|false
  */
-function imap_status($stream_id, string $mailbox, int $options) {}
+function imap_status($imap, string $mailbox, int $flags) {}
 
-/** @param resource $stream_id */
-function imap_mailboxmsginfo($stream_id): stdClass {}
+/** @param resource $imap */
+function imap_mailboxmsginfo($imap): stdClass {}
 
-/** @param resource $stream_id */
-function imap_setflag_full($stream_id, string $sequence, string $flag, int $options = 0): bool {}
+/** @param resource $imap */
+function imap_setflag_full($imap, string $sequence, string $flag, int $options = 0): bool {}
```

Comment:
Dilemma: This has an integer flags argument, but also a `$flag` argument ... I kept the `$options` name to avoid confusion.

Reply:
Doesn't seem like a big problem, so I agree with the approach taken